.gitignore

```diff
@@ -9,3 +9,5 @@ pom.xml.asc
 /.lein-*
 /.nrepl-port
 /.cpcache
+/src/medley/bin
+/src/medley/obj
```

Comment:
Those were the default output directories for building/packaging the project for NuGet. These probably aren't needed in the `.gitignore`, especially since there isn't an automatic build/package command provided from the root directory, so building or packaging for NuGet involves manually calling the appropriate commands from the `src/medley` directory.

Reply:
Those were the default output directories for building/packaging the project for NuGet. These probably aren't needed in the `.gitignore`, especially since there isn't an automatic build/package command provided from the root directory, so building or packaging for NuGet involves manually calling the appropriate commands from the `src/medley` directory.